arch/riscv/boot/dts/bouffalolab/bl808.dtsi

```diff
@@ -140,5 +140,11 @@
 			#interrupt-cells = <2>;
 			riscv,ndev = <64>;
 		};
+
+                wdt: wdt@2000a500 {
+                        compatible = "bflb,bflb808-wdt";
```

Comment:
Did you mean `bflb,bl808-wdt` here? There is no bflb808 chip.

Reply:
Updated in commit [7da84c3](https://github.com/arm000/linux-bl808/pull/8/commits/7da84c39ae0fa0616aae048c955a74efdbf80824)